src/gg.regression.unity.bots/Runtime/Prefabs/RGOverlay/DeleteSequenceDialog.prefab

```diff
@@ -738,19 +737,6 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   m_HorizontalFit: 0
   m_VerticalFit: 0
---- !u!114 &9089944729806318628
```

Comment:
Which behaviour is removed here ?

Reply:
I had a unrelated script attached `RGSaveSequenceButton` on the Delete button. Not useful